lib/make/makefile.bowtie2

```diff
@@ -70,6 +72,11 @@ all_interleaved_sorted_bams: $(subst .int.fastq.gz,.int.bowtie2.sorted.bam,$(wil
 	bowtie2 $(BOWTIE_ALIGN_OPTS) -x $(BOWTIE_INDEX) -1 $(word 1,$^) -2 $(word 2,$^) 2> $@.err | samtools view -Sb | samtools sort > $@
 	@echo "`date +"%Y%m%d %H:%M:%S"`: DONE" >> $@.makecall
 
+%.pe.bowtie2.sorted.bam: %.r1.fastq %.r2.fastq
```

Comment:
You might have a use case for this, but I avoid working with unzipped fastq files as much as I can and since bowtie2 handles zipped files, I don't see why one needs this. Better to zip any files that are not zipped before starting the mapping.

Reply:
Rules and targets with unzipped FASTQ files will be removed.